### DISCUSSION TITLE
MATsim version compatible with NoiseModelling

### DISCUSSION THREAD

**Discussion by lyselott** [Universite-Gustave-Eiffel/NoiseModelling#580] [Q&A] [answered]
Hello, I am trying to use MATsim and NoiseModelling in the same computer (Mac) but java 11 necessary to run NoiseModelling 4.0.3 is not enough to run MATsim 15.0 (java 18 is needed). So anyone you know what is the better older version of MATsim to use with NoiseModelling?

**Comment by nicolas-f** [comment, marked as answer]
Hi,

NoiseModelling use matsim version 11:

https://github.com/Universite-Gustave-Eiffel/NoiseModelling/blob/4.X/wps_scripts/build.gradle#L123
